onnxruntime/core/optimizer/compute_optimizer/shared_utils.h

```diff
@@ -134,5 +134,27 @@ Node* InsertIntermediateNodeOnDestInput(Graph& graph,
                                         const std::string& domain,
                                         const logging::Logger& logger);
 
+enum class DimCompareRet {
```

Comment:
Can we give a better name to this enum? `DimCompare` might also work instead of `DimCompareRet`.

Reply:
Done